README.md

```diff
@@ -16,9 +16,11 @@ dcos config prepend package.sources https://github.com/mesosphere/universe/archi
 
 ## Branches
 
-The default branch for this repository is `version-1.x`, which reflects the current schema for the Universe. In the future, if the format changes significantly, there will be additional branches.
+The default branch for this repository is `version-2.x`, which reflects the current schema for the Universe. In the future, if the format changes significantly, there will be additional branches.
 
-The `cli-tests` branch is used for integration testing by the [DCOS CLI](https://github.com/mesosphere/dcos-cli) and provides a fixed and well known set of packages to write tests against.
+This branch (version-1.x) has been deprecated, please use branch version-2.x instead.
```

Comment:
This and the line below are confusing. 

I'd merge them into one sentence and bump it right up to the top of the README in bold or something.

"This branch (`version-1x`) uses a deprecated Universe schema and can only be used by legacy versions of the DCOS CLI (less than 0.4.0). To ensure you can continue to install the latest packages, please [upgrade your installed DCOS CLI](https://docs.mesosphere.com/install/cli/)."